packages/components/src/ui/context/wordpress-component.ts

```diff
@@ -18,6 +18,7 @@ export type WordPressComponentProps<
 	Omit< React.ComponentPropsWithRef< T >, 'as' | keyof P | 'children' > &
 	( IsPolymorphic extends true
 		? {
+				/** The HTML element to render the component as. */
```

Comment:
The `as` prop can be also used to render a component as another component (not just an HTML element) — should we mention it in the comment?

Reply:
Good point, fixed.